src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml

```diff
@@ -370,7 +370,7 @@
     <sys:String x:Key="Toolbox_General_Tools">Allgemeine Werkzeuge</sys:String>
     <sys:String x:Key="Toolbox_Query_Office_Version">Office-Version abfragen</sys:String>
     <sys:String x:Key="Toolbox_Query_Office_Version.Description">Abfrage der Office-Update-Kanäle von Microsoft.</sys:String>
-    <sys:String x:Key="Toolbox_Query_Office_Version_GoBack">Back to toolbox page.</sys:String>
+    <sys:String x:Key="Toolbox_Query_Office_Version_GoBack">Zurück zur Toolbox-Seite.</sys:String>
```

Comment:
Should be "Zurück zur Werkzeug-Seite", because I translated "Tool" (for Toolbox pages) with "Werkzeug".

Reply:
Manche belassen es auch bei Toolbox 🤷‍♂️😉